code/modules/mob/living/simple_animal/abnormality/_auxiliary_modes/joke/aleph/sukuna.dm

```diff
@@ -0,0 +1,247 @@
+#define MALEVOLENT_SHRINE_COOLDOWN (60 SECONDS)
+/mob/living/simple_animal/hostile/abnormality/sukuna
+	name = "Sukuna"
+	desc = "The Heian Era asspuller himself."
+	icon = 'ModularTegustation/Teguicons/tegumobs.dmi'
+	icon_state = "sukunad"
+	icon_living = "sukunad"
+	portrait = "sukuna"
+	del_on_death = TRUE
+	maxHealth = 13000
+	health = 13000
+	var/can_act = TRUE
+	var/list/survivors = list()
+	var/cleave_cooldown
+	var/cleave_cooldown_time = 6 SECONDS
+	var/cleave_damage = 150
+	var/shrine_cooldown
+	var/shrine_cooldown_time = 60 SECONDS
+	var/shrine_damage = 900
+	var/current_stage = 1
+	ranged = TRUE
+	rapid_melee = 2
+	melee_queue_distance = 2
+	move_to_delay = 2.2
+	damage_coeff = list(RED_DAMAGE = 0.8, WHITE_DAMAGE = 0.5, BLACK_DAMAGE = 1.3, PALE_DAMAGE = -1) //funny
+	melee_damage_lower = 75
+	melee_damage_upper = 100
+	melee_damage_type = PALE_DAMAGE
+	attack_sound = 'sound/weapons/ego/da_capo2.ogg'
+	attack_verb_continuous  = "dismantles"
+	attack_verb_simple = "dismantle"
+	friendly_verb_continuous = "stares at"
+	friendly_verb_simple = "stare at"
+	faction = list()
+	can_breach = TRUE
+	threat_level = ALEPH_LEVEL
+	start_qliphoth = 1
+	// Work chance fluctuates based on level. left to right as level increase.
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(0, 0, 0, 0, 60),
+		ABNORMALITY_WORK_INSIGHT = list(0, 0, 0, 0, 30),
+		ABNORMALITY_WORK_ATTACHMENT = -1000, //lol, lmao
+		ABNORMALITY_WORK_REPRESSION = list(0, 0, 0, 30, 55),
+	)
+	work_damage_amount = 25
+	work_damage_type = RED_DAMAGE
+	chem_type = /datum/reagent/abnormality/violence
+	death_message = "dies, probably."
+	wander = TRUE
```

Comment:
```suggestion
	wander = TRUE
	can_spawn = FALSE
```
can_spawn has not been set to false. This means that if merged, this abnormality will spawn during normal rounds. You can leave this on for a test-merge, but you must eventually do this.

Reply:
ok